### DISCUSSION TITLE
Cap Refresh rate at 60HZ for osu! taiko

### DISCUSSION THREAD

**Discussion by maxie5353** [ppy/osu#23091] [Polls]
Currently HR within osu taiko is put behind a paywall of being able to afford a higher refresh rate monitor. While it is not technically necessary to own a high refresh rate monitor, the common census among top players is that having a higher refresh rate monitor (such as 144HZ) provides an inherent advantage compared to playing on 60HZ. 

I am not too sure on the technical side of this aspect but having a higher refresh rate makes the notes look smoother and you see them more of the note in the same time it takes to have it go across the screen. This provides a massive advantage at higher scroll speeds within taiko as there is a very short amount of time to actually view the note, so having it be smoother makes HR way simpler.

The main reason I am polling this question is because I believe that there should not be a paywall that allows for easier playing of HR. This is different than better computers running osu better, as this is a clear cut advantage for being able to afford a monitor that provides that advantage. For these reasons I think that the HZ rate for osu!taiko should be capped at 60.

I would also like to add that I do not know the practicality of this change (if its even possible) and that I have never used github before so I apologize for anything being formatted poorly or if I am breaking etiquette.

**Comment by bdach** [comment]
taiko players [intentionally want to have the playfield display a different range of notes depending on screen resolution](https://github.com/ppy/osu/discussions/13047), and this has been a consistent point of outcry. I don't imagine capping to 60fps is going to go over well with them.

On top of that, then why not apply that cap to all rulesets? Or, if we continue that logic, why aren't all competitive multiplayer games ever capped to 60fps for parity?

I don't think this is a reasonable proposal.

**Reply by maxie5353** [reply]
I have gotten feedback from other players as well and this seems to be a universally disliked proposal so you can resolve this (if that is a thing)

**Comment by peppy** [comment]
You can do this by setting vsync at your drivers and changing your system refresh rate if you wish.

Forcefully limiting this for a user is anti-user and not even worth polling.
